support-frontend/assets/pages/contributions-landing/components/ContributionForm.tsx

```diff
@@ -138,14 +71,14 @@ const mapStateToProps = (state: State) => ({
 			state.page.checkoutForm.personalDetails.email,
 			state.page.user.email,
 		) ?? '',
-	otherAmounts: state.page.form.formData.otherAmounts,
+	otherAmounts: state.page.checkoutForm.product.otherAmounts,
 	paymentMethod: state.page.form.paymentMethod,
 	existingPaymentMethod: state.page.form.existingPaymentMethod,
 	stripeClientSecret:
 		state.page.form.stripeCardFormData.setupIntentClientSecret,
-	contributionType: state.page.form.contributionType,
+	contributionType: getContributionType(state),
```

Comment:
Minor point, but we call `getContributionType(state)` three times here, maybe we could call it once and assign the return value to a variable?